libs/cardano-ledger-core/src/Cardano/Ledger/Core/TxCert.hs

```diff
@@ -152,6 +156,19 @@ instance NoThunks (PoolCert c)
 instance NFData (PoolCert c) where
   rnf = rwhnf
 
+instance Crypto c => ToJSON (PoolCert c) where
+  toJSON = \case
+    RegPool poolParams ->
+      object
```

Comment:
We could use 
`kindObject "RegPool" [ "poolParams" .= toJSON poolParams]`  here  I think (not that it makes a difference)

Reply:
Yes. Done